dandi/bids_validator_xs.py

```diff
@@ -581,31 +604,28 @@ def select_schema_dir(
             else:
                 with open(dataset_description) as f:
                     dataset_info = json.load(f)
-                    if force_select:
-                        try:
-                            schema_version = dataset_info["BIDSVersion"]
-                        except KeyError:
-                            return utils.get_schema_path()
-                    else:
+                    try:
                         schema_version = dataset_info["BIDSVersion"]
+                    except KeyError:
+                        schema_version = schema_min_version
+        if schema_min_version:
+            if schema_version < schema_min_version:
+                schema_version = schema_min_version
```

Comment:
re using semver (needs import) and warning discussed in https://github.com/dandi/dandi-cli/issues/950#issuecomment-1101174851

```suggestion
            if Version(schema_version) <= Version(schema_min_version):
                if schema_version != schema_min_version:
                    lgr.warning("BIDSVersion %s is less than the smallest known %s, so we will use the known one",
                        schema_version, schema_min_version)
                schema_version = schema_min_version
```